### PR TITLE
Merge /careers and /team into /who-we-are

### LINKED ISSUE #67: Add origin story (for example at publiccode.net/history)
Why: @lscheske and @nebster believe the origin page from the case for support will increase trust from potential member orgs

Timing: could be (roughly) done in honor of the first anniversary of our founding (3 June 2019)

Related page: https://publiccode.net/team/
Related issue: #66

### PR DESCRIPTION
**Pull request by ElenaFdR**
Closes #67.

Since we're only passively recruiting atm and careers are also mentioned in our footer, deprioritizing the careers page makes sense.

Still to do on this PR:
- [x] tweak the Foundation history so it makes more sense for a general audience (big Q: do Arnout and Pieter want to be credited in this way? So far their contributions are harder to find out about).
- [x] redirects from all old URLs to new URLs for /careers and /team .md files
- [x] Q: how do you add a redirect to a .jpg???
- [x] add /who-we-are to publiccode.net/index, remove /careers and /team
- [x] update any random links to team or careers across publiccode.net
- [x] update URL for /careers and /team on the upper footer
- [x] fix liquid for bio photos rendering (see note below)
- [x] update URL for /careers on the lower footer (in the Jekyll theme) ([PR #43](https://github.com/publiccodenet/jekyll-theme/pull/43))
- [x] update about.publiccode.net to update all relevant links to either team or careers ([PR #754](https://github.com/publiccodenet/about/pull/754))

-----
[View rendered who-we-are/careers/community.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/careers/community.md)
[View rendered who-we-are/careers/index.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/careers/index.md)
[View rendered who-we-are/careers/intern.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/careers/intern.md)
[View rendered who-we-are/careers/product-marketing.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/careers/product-marketing.md)
[View rendered who-we-are/careers/quality.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/careers/quality.md)
[View rendered who-we-are/careers/support.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/careers/support.md)
[View rendered who-we-are/index.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/index.md)
[View rendered who-we-are/team/alba-roza.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/alba-roza.md)
[View rendered who-we-are/team/ben-cerveny.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/ben-cerveny.md)
[View rendered who-we-are/team/boris-van-hoytema.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/boris-van-hoytema.md)
[View rendered who-we-are/team/claus-mullie.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/claus-mullie.md)
[View rendered who-we-are/team/deborah-meibergen.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/deborah-meibergen.md)
[View rendered who-we-are/team/elena-findley-de-regt.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/elena-findley-de-regt.md)
[View rendered who-we-are/team/eric-herman.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/eric-herman.md)
[View rendered who-we-are/team/felix-faassen.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/felix-faassen.md)
[View rendered who-we-are/team/jan-ainali.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/jan-ainali.md)
[View rendered who-we-are/team/laura-scheske.md](https://github.com/publiccodenet/publiccode.net/blob/Who-we-are/who-we-are/team/laura-scheske.md)

### CODE REVIEW

**Comment by ElenaFdR** [comment]
I could add the  /team .jpg redirects to the person's .md file, since that has frontmatter? It wouldn't completely break the user journey (but might confuse search engines that previously indexed our photos?)

**Comment by ElenaFdR** [comment]
Re: fix liquid for bio photo render on who-we-are/index

The 2 bits of liquid that are currently broken are:
- _layouts/bio.html
- who-we-are/index.md

None of the bio images is currently rendering correctly on either:
- who-we-are/index.md
- individual bio pages (like who-we-are/team/alba-roza.md)

Since the person.jpg files are still in the same folder with the person.md files, I'm not sure why they're not rendering. Perhaps some bit of the liquid that doesn't look like a folder structure still resolves as /team and needs to be updated to the equivalent ../team?  There are lots of #team references in the list.

**Comment by ElenaFdR** [comment]
VScode gives me this error: 

[2020-09-25 12:42:24] ERROR `/who-we-are/alba-roza.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/ben-cerveny.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/deborah-meibergen.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/claus-mullie.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/boris-van-hoytema.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/elena-findley-de-regt.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/eric-herman.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/felix-faassen.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/laura-scheske.jpg' not found.
[2020-09-25 12:42:24] ERROR `/who-we-are/jan-ainali.jpg' not found.

So I think I'm right about the liquid looking for photos in the same folder for who-we-are/index.md. Doesn't explain why it's not loading on individual pages.

**Comment by ElenaFdR** [comment]
Using dev tools to look at the rendered version, it looks like the bio template isn't being applied to individual bio pages (none of the frontmatter is rendering - so no title and no image).

Logging things I've tried to fix this:

- removing the image redirect and updating the path (line 23) in _config so that the bio layout is applied to everything in the /team folder (no change)
- in _config, changing path to ```type: "bio"```

**Comment by ElenaFdR** [comment]
I've decided to leave the Foundation's history as it appears in the case for support for transparency's sake - it's a nice way to explain the range of expertise/support we can draw on.

File: who-we-are/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - team/index.md
+---
+
+# Meet us
+
+The Foundation for Public Code was founded by Ben Cerveny, Boris van Hoytema and Arnout Schuijff in 2019 to guide international collaboration among public organizations around codebase design, quality and governance.
+
+Leveraging 30 combined years of experience working with cities and technology-enabled collaboration, Boris and Ben recognized policy and whitepapers were ineffective catalysts in this space. Their landscape analysis revealed that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation.
+
+Based on their experience of technological innovation in a traditional industry, Arnout and Pieter van der Does (co-founders of the Adyen payments platform) agreed with Ben and Boris that public organizations deserved better software. They joined the Board of Directors to share their expertise and connections.
```
Comment (Ainali) [inline who-we-are/index.md:14]: I know we introduced Arnout earlier in the text, but reading it quickly, this sounds like the van der Does brothers.

File: who-we-are/careers/index.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+redirect_from:
+    - careers/index.md
+---
+
 # Careers
 
 Join our mission to enable public-purpose software and policy that is open and collaborative.
```
Comment (lscheske) [inline who-we-are/careers/index.md:8]:
```suggestion
Join us in enabling public-purpose software and policy that is open and collaborative.
```

File: who-we-are/index.md (hunk 1 above)
Comment (lscheske) [inline who-we-are/index.md:14]:
```suggestion
Based on their experience of technological innovation in a traditional industry, Pieter van der Does and Arnout Schuijff (co-founders of the Adyen payments platform) agreed with Ben and Boris that public organizations deserved better software. They joined the Board of Directors to share their expertise and connections.
```

**Review by lscheske — CHANGES_REQUESTED** [review]
I've made some minor changes:
* editorial change in the intro text
* editorial change based on Jan's comment

File: who-we-are/index.md (hunk 1 above)
Comment (ElenaFdR) [reply who-we-are/index.md:14]: @lscheske Arnout's mentioned higher up the page - I would normally only introduce someone with their full name once. Do you think people will miss the previous reference?

**Comment by bvhme** [comment]
I miss the documentation on _why_ we're changing all these links and the file structures.

File: who-we-are/index.md (hunk 1 above)
Comment (lscheske) [reply who-we-are/index.md:14]: This is true, but I also see @Ainali's point about the last name seemingly applying to both in this sentence. 

I have an alternative solution: legally speaking ([Articles of Association](https://about.publiccode.net/organization/articles-of-association.html), Pieter is also a co-founder of the Foundation (4th founder). So we could mention him by full name in line 10, then in this line (14) we only have to write Pieter & Arnout. What do you think?

Comment (ElenaFdR) [reply who-we-are/index.md:14]: Ah, I hadn't spotted Jan's comment, as it was resolved.

**Comment by ElenaFdR** [comment]
@bvhme As the pull request says, I'm doing this to remove duplicate links from the homepage. Careers is already covered in the footer. I can create an issue to document this, if you think that's necessary.

**Review by lscheske — APPROVED** [review]

**Comment by bvhme** [comment]
I'm saying this because depending on our success this might be the last time we ever want to change those URLs, as it might happen that we want to, for instance, print the URL on leaflets, tell people to go to that URL in podcasts etcetera or something or link it from a whole lot of other places.

I foresee us trying to figure out why we made this decision in the future when we say 'just check public code dot net slash who dash we dash are slash careers' to someone at an event.

(The structure for the end user navigating the site does not need to be equated in the paths and URLs, those can be different and serve different purposes.)

**Comment by ElenaFdR** [comment]
@bvhme This makes sense, but I wish you had mentioned this earlier.

**Comment by bvhme** [comment]
I'm not trying to block this, just saying that maybe you can update the pull request content to reflect this decision so that it'll be on top in the future and we know. Then I think I can approve this pull request.

**Comment by ElenaFdR** [comment]
@bvhme Can this need for snappy URLs be met via [permalinks](https://jekyllrb.com/docs/permalinks/), rather than disassembling the file trees?

**Comment by bvhme** [comment]
Of course, it adds complexity that I chose to avoid in order to make it easier to contribute, however you might deem it worth it here.

**Comment by ElenaFdR** [comment]
@bvhme This is ready for your re-review. I've moved /careers out of of /who-we-are, so we'll still have access to that URL in the future.

**Comment by ElenaFdR** [comment]
Errors found so far in #99 to justify reverting it:

- [ ] duplicate bio pages for @demkodo
- [ ] none of the bio redirects are working
- [ ] @AlbaRoza's bio doesn't have redirects on it

**Pull request merged**